src/main/java/org/broadinstitute/consent/http/models/DarDecisionMetrics.java

```diff
@@ -113,6 +120,28 @@ private void setDatasetId(DataSet dataset) {
     }
   }
 
+  private void setCountUniqueUser(DataAccessRequest dar) {
+    List<String> emails = null;
+    if (Objects.nonNull(dar.data)) {
```

Comment:
We should always call the getter and not pull the field directly.
```suggestion
    if (Objects.nonNull(dar.getData())) {
```